docker-fedora/init.sh

```diff
@@ -1,15 +1,13 @@
-#!/bin/bash
+#!/bin/sh
 
 # set storage first
 (
-    . /etc/sysconfig/docker-storage-setup
-    /usr/bin/docker-storage-setup
+        . /etc/sysconfig/docker-storage-setup
```

Comment:
no not really, I had just indented the whole file after I worked on it, I'll fix that